docs/api-guide.md

```diff
@@ -729,21 +725,21 @@ public class Sample {
         // Commit the transaction
         tx.commit();
       } catch (CrudConflictException | CommitConflictException e) {
-        // If you catch CrudConflictException or CommitConflictException, it indicates conflicts 
-        // happen during a transaction so that you can retry the transaction
+        // If you catch CrudConflictException or CommitConflictException, it indicates a conflict 
+        // occurs during the transaction so that you can retry the transaction from the beginning
         try {
-          tx.abort();
-        } catch (AbortException ex) {
-          // Aborting the transaction fails. You can log it here
+          tx.rollback();
+        } catch (RollbackException ex) {
+          // Rolling back the transaction fails. You can log it here
         }
         retryCount++;
       } catch (CrudException | CommitException e) {
         // If you catch CrudException or CommitException, it indicates some failure happens, so you
         // should cancel the transaction or retry the transaction after the failure/error is fixed
         try {
-          tx.abort();
-        } catch (AbortException ex) {
-          // Aborting the transaction fails. You can log it here
+          tx.rollback();
+        } catch (RollbackException ex) {
+          // Rolling back the transaction fails. You can log it here
```

Comment:
> Rolling back the transaction fails
`failed` sounds a bit better?

Reply:
Fixed in 1041dda610d74ac00596e1ab1f69d6bbdeb32106.